test/analysis/knockouts.jl

```diff
@@ -1,20 +1,20 @@
 @testset "single_knockout" begin
     m = StandardModel()
-    add!(m, Metabolite("A"))
-    add!(m, Metabolite("B"))
+    add_metabolites!(m, Metabolite("A"))
```

Comment:
(here the singular `add_metabolite` would look much more reasonable)

(nitpicking)

Reply:
yep, I also think so. The only reason I had it like this is because in `CoreModel` this happens: 
```julia
function remove_reactions(m::CoreModel, rxn::Integer)
    return remove_reactions(m, [rxn])
end
```

hmm, we should pick one style though... I don't mind too much, you decide?

Reply:
went with singular/plural everywhere